docs/source/developing/writing-docs.adoc

```diff
@@ -12,85 +12,156 @@ toc::[]
 
 This section contains information about contributing to the Minishift documentation.
 
-[[section-building-minishift-docs]]
-== Building Minishift documentation
+[[contribute-to-docs]]
+== Contributing to the documentation
 
-Minishift documentation is located in the `docs` sub-folder. The
+Minishift is an open-source project and we welcome contributions. Similar to code
+contributions, if you want to add or edit Minishift documentation, you can create an
+issue in our link:https://github.com/minishift/minishift/issues[Github issue tracker] and
+submit a pull request with the changes or request assistance from our technical writers.
+
+[[docs-conventions-guidelines]]
+== Conventions and guidelines
+
+The Minishift documentation is authored in the AsciiDoc markup format. Since Minishift is
+closely related to the OpenShift Origin project, we follow the same conventions for
+tagging, formatting, and structure.
+
+The documentation source structure follows a modular format, where each file is called
+a _topic_ and each topic is stored under a sub-folder based on the relevant category, such as
+getting started, using Minishift, or command reference.
+
+TIP: You can check out the raw file for this topic or other existing Minishift topics to see
+how information is structured and which tags to use.
+
+[[common-conventions]]
+=== Commonly-used conventions
+
+- Each file must contain a mandatory header section with metadata that is required for publication.
```

Comment:
I think this still needs clarification. What is mandatory? Or we relax the sentence to:

"Each file starts with a level one heading, followed by any necessary asciidoc variable declarations."